pkg/configui/configui.go

```diff
@@ -347,26 +346,25 @@ func SetTheme(theme *tview.Theme) {
 	theme.ContrastBackgroundColor = tcell.ColorFuchsia
 }
 
-func ConfigUiBuilder(cfg *config.Instance, pl platforms.Platform) (*tview.Application, error) {
-	app := tview.NewApplication()
-	pages := tview.NewPages()
-
-	SetTheme(&tview.Styles)
-
-	BuildMainMenu(cfg, pages, app)
+func ConfigUiBuilder(cfg *config.Instance, app *tview.Application, pages *tview.Pages, exitFunc func()) (*tview.Application, error) {
+	BuildMainMenu(cfg, pages, app, exitFunc)
 	BuildTagsMenu(cfg, pages, app)
 	BuildTagsReadMenu(cfg, pages, app)
 	BuildTagsWriteMenu(cfg, pages, app)
 	BuildAudionMenu(cfg, pages, app)
 	BuildReadersMenu(cfg, pages, app)
 	BuildScanModeMenu(cfg, pages, app)
-	pages.SwitchToPage("main")
+	pages.SwitchToPage("mainconfig")
 
 	return app.SetRoot(pages, true).EnableMouse(true), nil
 }
 
 func ConfigUi(cfg *config.Instance, pl platforms.Platform) error {
 	return BuildAppAndRetry(func() (*tview.Application, error) {
-		return ConfigUiBuilder(cfg, pl)
+		app := tview.NewApplication()
+		pages := tview.NewPages()
```

Comment:
app and pages are now created from the config, so that ConfigUiBuilder can be used to append the pages to an existing app